raiden-dapp/src/locales/en.json

```
@@ -9,6 +9,54 @@
     "persist-checkbox": "Don't display this message on start up",
     "accept-button": "Accept"
   },
+  "info-overlay": {
+    "done": "Close",
+    "transfer": {
+      "header-image": "transfer_info",
+      "header": "About Transfer",
+      "body": "Enter a recipient address and a token amount to make a transfer. You need to have an open channel with the recipient or with a hub that is connected to the recipient."
+    },
+    "transfer-steps": {
+      "header-image": "transfer_steps_info",
+      "header": "About Direct and Mediated Transfers",
+      "body": "A direct transfer will be possible if you have an open channel with the recipient. In any other case, the PFS will try to find the best way to route your transfer to the recipient."
+    },
+    "select-hub": {
+      "header-image": "select_hub_info",
+      "header": "About Selecting a Hub",
+      "body": "A Hub is a well connected node. You can connect to a hub or to any other node. Make sure you have enough service tokens as well as a balance of your selected token."
+    },
+    "open-channel": {
+      "header-image": "open_channel_info",
+      "header": "About Opening a Channel",
+      "body": "Opening a channel will connect you to your selected hub or node. You need to choose an amount to deposit in the channel before opening it."
```

```suggestion
      "body": "Opening a channel will connect you to your selected hub or node. You need to choose an amount to deposit in the channel. Later you can transfer small amounts of this deposit to any recipient in the network."   
```
Not sure, if this is well phrased